modules/openapi-generator/src/test/java/org/openapitools/codegen/testutils/IntegrationTestPathsConfig.java

```diff
@@ -17,11 +17,14 @@
 
 package org.openapitools.codegen.testutils;
 
+import java.net.URISyntaxException;
+import java.net.URL;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.Objects;
 
 public class IntegrationTestPathsConfig {
-    private static final Path INTEGRATION_TEST_PATH = Paths.get("target/test-classes/integrationtests").toAbsolutePath();
+    private static final Path INTEGRATION_TEST_PATH = getIntegrationTestsPath();
```

Comment:
Previously this called into the Maven `target` directory which is not going to be there in Gradle. This now uses a method of loading the integration test resources in a manor which will work for both build tools.